pkg/config/config.go

```diff
@@ -144,6 +153,16 @@ type Sources struct {
 	DefaultMessage string `toml:"default_message"`
 }
 
+// Client are the config options for the client.
+type Client struct {
+	KeycloakUrl      string `toml:"keycloak_url"`
+	KeycloakRealm    string `toml:"keycloak_realm"`
```

Comment:
If you add json struct tags you can simply send the structure as JSON in the endpoint.

Reply:
Done